src/test/java/com/github/mkopylec/sessioncouchbase/TestConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.github.mkopylec.sessioncouchbase;
+
+import org.apache.catalina.Context;
+import org.apache.tomcat.util.http.LegacyCookieProcessor;
+import org.springframework.boot.context.embedded.ConfigurableEmbeddedServletContainer;
+import org.springframework.boot.context.embedded.EmbeddedServletContainerCustomizer;
+import org.springframework.boot.context.embedded.tomcat.TomcatContextCustomizer;
+import org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedServletContainerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class TestConfiguration {
```

Comment:
Just a tip for me :)
Could you please let me know when this bug will be fixed so I can remove this class?